apps/prairielearn/src/lib/editors.ts

```diff
@@ -509,7 +509,8 @@ export abstract class Editor {
     const idSplit = id.split(path.sep);
 
     // Start deleting subfolders in reverse order
-    const reverseFolders = idSplit.slice(0, -1).toReversed();
+    // eslint-disable-next-line unicorn/no-array-reverse
+    const reverseFolders = idSplit.slice(0, -1).reverse();
```

Comment:
This was needed because our various `tsconfig` files don't agree on `lib`:

- `apps/prairielearn/assets/scripts/tsconfig.json` (for client-side code) uses `ES2022`
- `packages/tsconfig/tsconfig.base.json` (default, ends up being used for server-side code) uses `ES2024`

`toReversed()` is part of `ES2023`, and thus can't be used in code that's type-checked under client rules; but we have a lint rule that mandates that `toReversed()` be used instead of `reversed()`. Something has to change here.